vars/utils.groovy

```diff
@@ -1,11 +1,16 @@
 def cosaCmd(params = [:]) {
 // Available parameters:
-//     cosaDir:        string  -- cosa working directory
 //     args:           string  -- arguments to pass to `cosa`
+//     cosaDir:        string  -- cosa working directory
+//     stage:          string  -- CI stage name
 //     user:           string  -- user to run the cosa commands 
     def cosaDir = getCosaDir(params)
     def args = params.get('args', "");
 
+    if (params['stage']) {
+        stage(params['stage'])
```

Comment:
Same suggestion here as in https://github.com/coreos/coreos-ci-lib/pull/52#discussion_r532880137.